MAINTAINERS

```diff
@@ -3,4 +3,6 @@ Dmitry Sorokin <dmitriy.sorokin@zalando.de>
 Ricardo De Cillo <ricardo.de.cillo@zalando.de>
 Valentine Gogichashvili <valentine.gogichashvili@zalando.de>
 Viktor Buldakov <viktor.buldakov@zalando.de>
```

Comment:
I think he is not maintainer anymore, at least his email should have been deactivated